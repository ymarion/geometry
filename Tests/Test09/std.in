# Creating a circle
C c1 0 10 10
# And now, moving it
MOVE c1 10 10
# Displaying the moved circle
LIST
#
EXIT
