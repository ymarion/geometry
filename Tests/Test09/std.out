# Creation
OK
# MOVE
OK
# LIST
C c1 10 20 10
